src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -113,6 +121,21 @@ public function getParameterBag()
         return $this->parameterBag;
     }
 
+    public function setEnvironmentBag(EnvironmentBagInterface $environmentBag)
+    {
+        $this->environmentBag = $environmentBag;
+    }
+
+    public function getEnvironmentBag()
+    {
+        return $this->environmentBag;
+    }
```

Comment:
Could it be this? https://github.com/symfony/symfony/pull/18155/files#diff-49d280a92634428dd8ab71cb611e37acR44